docs/platforms/dotnet/guides/maui/enriching-events/screenshots/index.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: "Screenshots"
+description: "Learn more about taking screenshots when an error occurs. Sentry pairs the screenshot with the original event, giving you additional insight into issues."
+---
+
+When a user experiences an error, an exception or a crash, Sentry provides the ability to take a screenshot and include it as an <PlatformLink to="/enriching-events/attachments/">attachment</PlatformLink>.
+
+This feature only applies to SDKs with a user interface, such as the ones for mobile and desktop applications. In some environments like native iOS, taking a screenshot requires the UI thread and in the event of a crash, that might not be available. Another example where a screenshot might not be available is when the event happens before the screen starts to load. So inherently, this feature is a best effort solution.
+
+## Enabling Screenshots
+
+Screenshots may contain <PlatformLink to="/data-management/sensitive-data/">PII</PlatformLink> and is an opt-in feature. You can enable it as shown below:
+
+<PlatformContent includePath="enriching-events/attach-screenshots" />
+
+Capturing a screenshot is a synchronous operation supported on both iOS and Android, with the requirement of using the UI thread specifically on iOS.
+
+## Viewing Screenshots
+
+A thumbnail of the screenshot shows up at the top of the **Issue Details** page:
+
+![Screenshot Thumbnail](/platforms/dotnet/guides/xamarin/enriching-events/screenshots/screenshot-thumbnail.png)
```

Comment:
Would it be possible to make the screenshot a bit higher resolution? This one is a bit fuzzy. Since we've moved to Next.js, all images need to be added to the public folder :) https://docs.sentry.io/contributing/pages/images/

Reply:
Yes, definitely! I'll update them then.